### DISCUSSION TITLE
How were the splits for the coq files decided per project?

### DISCUSSION THREAD

**Discussion by brando90** [princeton-vl/CoqGym#79] [Q&A] [answered]
I was curious & wanted to extend the data set size of coq-gym -- ideally automatically to include a lot. Were the files to do the splits decided manually? Or randomly? or maybe Coq was used to create a dag of dependencies and then a topological sort & then some arbitrary split based on the dag was made?

Thanks in advance! :)

**Comment by yangky11** [comment, marked as answer]
The projects are randomly split into train/val/test. All files within the same project belong to the same split.

**Reply by brando90** [reply]
ok but you didn't split per file I see! Is that to avoid the complication of the depedency dag thing? @yangky11

**Reply by yangky11** [reply]
We didn't want a test theorem to be accessible in the environment of a training theorem.
